css/orderpage.css

```diff
@@ -0,0 +1,184 @@
+
+ :root {
+    --grey-dark: #848993;
+    --grey-darker: #4D5057;
+    --grey-light: #E5E5E5;
+    --orange-dark: #E0633A;
+    --orange-light: #FCEFEB;
+    --white: #fff;
+}
+main {
+    display:flex;
+    margin-left:7%;
+    min-height: 80vh;
+}
+main > section {
+    width: 50%;
+}
+.form-container {
+    margin: auto 7% auto 0; 
+}
+
+.image-container {
+    text-align: right;
+}
+.form-container h1 {
+    font-size: 3rem;
+    font-weight: 500;
+    color:var(--orange-dark);
+    padding-bottom:1rem;
+}
+form {
+    display: flex;
+    flex-direction: column;
+    color:var(--grey-darker);
+}
+label {
+    margin-bottom: 0.4rem;
+}
+form input,
+form select {
+    margin-bottom: 15px;
+    padding:5px;
+    border: 1px solid var(--grey-darker);
+    border-radius: 4px;
```

Comment:
Try to avoid px measurements, and use rem or another form of relative measurements where possible.